client/src/kb.c

```diff
@@ -388,6 +402,13 @@ const char * linux_to_display[KEY_MAX] =
   [KEY_F14]              = "F14",
   [KEY_F15]              = "F15",
   [KEY_PRINT]            = "Print",
+  [KEY_MUTE]             = "Mute",
+  [KEY_VOLUMEUP]         = "VolumeUp",
+  [KEY_VOLUMEDOWN]       = "VolumeDown",
+  [KEY_NEXTSONG]         = "NextSong",
+  [KEY_PLAYPAUSE]        = "PlayPause",
+  [KEY_PREVIOUSSONG]     = "PreviousSong",
+  [KEY_STOPCD]           = "StopMedia",
```

Comment:
I chose this name instead of STOPCD because it is easier to understand

Reply:
Seems fine to me